app/javascript/controllers/navigateable_table_controller.js

```diff
@@ -0,0 +1,16 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  connect() {
+    this.element.querySelectorAll("tr[data-href]").forEach((row) => {
+      row.setAttribute("data-action", `click->${this.identifier}#navigate`);
+    });
+  }
+
+  navigate(event) {
+    const url = event.currentTarget.dataset.href;
+    if (url) {
+      window.location.href = url;
+    }
+  }
+}
```

Comment:
in general great use of stimulus / vanilla js throughout this PR! Helping with the push against the tide of existing jquert